mountpoint-s3/scripts/network_performance.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Please ensure 'jq' and AWS CLI v2 are installed before executing this script.
+# The purpose of this script is to generate a json file contains network throughput numbers of each EC2 instance types.
+
+version=v0.0.1 # release version
+result_file=network_performance.json
+temp_file=$(mktemp)
+
+# query
+query_ec2_instance_network_throughput="aws ec2 describe-instance-types
+--filters \"Name=instance-type,Values=*\"
+--query \"InstanceTypes[].[InstanceType, NetworkInfo.NetworkPerformance]\"
+--region us-east-1
+--output json"
+
+(> ${mktmp})
```

Comment:
I think we need to replace all mentions of `mktmp` here.

Please drop the `network_performance.json` and check it all works OK for the next/final commit.

Reply:
we can't drop network_performance.json, otherwise `let file = include_str!("../scripts/network_performance.json");` results compile error.

Reply:
also, I think we need to balance the velocity vs 2-way-door customer experience.

Reply:
Sorry, I was meaning to suggest to replace the existing file with the one created by the script